sidebars.js

```diff
@@ -94,6 +94,13 @@ const sidebars = {
           'doc-code-samples',
         ],
       },
+      {
+        type: 'category',
+        label: 'Account',
+        collapsible: true,
+        collapsed: true,
+        items: ['account/user-account-settings', 'account/billing', 'account/github-student']
```

Comment:
@jayk-u this part was missing, so the pages weren't displayed in the sidebar. We need to double-check this file in each PR to be sure that nothing that should be displayed is missing.

Reply:
Nope, it's part of the writer's job to manually check that the changes are accessible on it's local running instance, nothing to automate here :)